### PR TITLE
rework readme a bit

### PR DESCRIPTION
**Pull request by grosser**
@byroot 
- bringing the speedup details up, cause that's why anyone is here ;)
- simplify Gemfile instructions (I hope ppl know how to bundle)
- setting a path that will work so copy-pasters do not get screwed
- must be -> should be, since it works well without that ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
-
-    $ bundle
-
 Then you need to add right after `require 'bundler/setup'`:
 
 ```ruby
 require 'bundler/setup'
 require 'bootscale'
-Bootscale.setup(cache_directory: '/path/to/cache/directory')
+Bootscale.setup(cache_directory: 'tmp/bootscale')
```
Comment (byroot) [inline README.md:21]: Actually I don't ensure the directory exist, so it wouldn't work for copy pasters. Maybe we just default it to `tmp/`?

File: README.md [hunk 2]
```diff
@@ -1,32 +1,31 @@
 # Bootscale
 
-Speedup applications boot by caching require calls
+Speedup applications boot by caching require calls.
+
+Spee gain depends on your number of gems. Under 100 gems you likely won't see the difference,
```
Comment (byroot) [inline README.md]: You have a typo here: `Spee`

**Comment by byroot** [comment]
A couple comments but :heart:

File: README.md (hunk 1 above)
Comment (RoxasShadow) [inline README.md:21]: I think saving the files just in `tmp/` can mess a little that folder. I suggest to keep `tmp/bootscale` and create the folder if it does not exist.

**Comment by grosser** [comment]
fixed typo, do we need a fix for when the folder does not exist ?

File: README.md (hunk 1 above)
Comment (byroot) [inline README.md:21]: Right we could do a FileUtils.mkdir_p

**Comment by grosser** [comment]
added mkdir

FYI master is super slow now takes boot time down to 50s from 20s ... so something bad is going on

**Comment by byroot** [comment]
> FYI master is super slow now takes boot time down to 50s from 20s ... so something bad is going on

That is weird, for me without:

```
real    0m16.781s
user    0m11.966s
sys 0m3.744s
```

and with bootscale:

```
real    0m12.565s
user    0m10.081s
sys 0m1.403s
```

Earlier today I was using bootscale as a local gem with `gem 'bootscale', path: '/path/to/bootscale'`, and for some weird reason the performance was horrible.

I just tried again an it's still the case:

```
real    0m22.905s
user    0m19.995s
sys 0m1.835s
```

I'm not sure what bundler is doing to be so slow.

**Comment by byroot** [comment]
Thanks for the patch! And let me know if you figure out what is killing the performance.

**Comment by grosser** [comment]
ah good, that must be it ...

**Pull request merged**